### PR TITLE
Add Rust interview with Luca Palmieri

### PR DESCRIPTION
**Pull request by emmasofiah2o**
- Add as blog post
- Embed video
- Add og-image
- Add visual

### CODE REVIEW

**Comment by emmasofiah2o** [comment]
The youtube link appears broken in the preview but works from the code. Not sure how to fix that

File: src/posts/2023-06-28-present-and-future-of-rust-with-luca-palmieri.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+authorHandle: marcoow
+tags: [rust]
+bio: "Marco Otte-Witte"
+description:
+  "Luca Palmieri and Marco Otte-Witte discuss the present and future of the Rust
+  Web Ecosystem."
+og:
+  image: /assets/images/posts/2023-06-28-present-and-future-of-rust-with-luca-palmieri/og-image.jpg
+tagline: |
+  <p>Rust has become more popular among web developers in recent years. In this interview, Luca Palmieri, Senior Software Engineer at Mainmatter and Rust author, and Marco Otte-Witte, Founder of Maimatter, delved into the world of backend development with Rust, examining the state of the ecosystem, the challenges it faces, and the success stories it generated.</p>
```
Comment (marcoow) [inline src/posts/2023-06-28-present-and-future-of-rust-with-luca-palmieri.md]:
```suggestion
  <p>Rust has become more popular among web developers in recent years. In this interview, Luca Palmieri, Principal Engineering Consultant at Mainmatter and Rust author, and Marco Otte-Witte, Founder of Maimatter, delved into the world of backend development with Rust, examining the state of the ecosystem, the challenges it faces, and the success stories it generated.</p>
```

File: src/posts/2023-06-28-present-and-future-of-rust-with-luca-palmieri.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+range of applications. As Rust continues to integrate into libraries and
+frameworks across ecosystems, developers increasingly reap the benefits of its
+power. Whether seeking predictable performance, improved scalability, or reduced
+infrastructure costs, Rust is a versatile language to consider when building
+high-performance and robust software solutions.
+
+Check out the full interview below and don’t hesitate to contact us to learn how
+your company could benefit from adopting Rust.
+
+<iframe width="560" height="315" src="https://youtu.be/Xb7NokhAVKI" title="Embedded video of Luca's interview" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (marcoow) [inline src/posts/2023-06-28-present-and-future-of-rust-with-luca-palmieri.md]: This doesn'T seem to work – check the preview

File: src/posts/2023-06-28-present-and-future-of-rust-with-luca-palmieri.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+Luca shared his personal experience with Rust adoption, starting with his work
+at TrueLayer, where he successfully transitioned from a predominantly C#
+environment to Rust, driven by the need for better maintainability, correctness,
+scalability, and latency. Despite initial concerns about hiring and lacking
+expertise, the project was a success, leading to enhanced job satisfaction,
+streamlined recruitment, and the creation of reliable and performant services.
+
+Luca then talked about his book "Zero to Production in Rust: An introduction to
+backend development in Rust" with which he aimed to bridge the gap between
```
Comment (marcoow) [inline src/posts/2023-06-28-present-and-future-of-rust-with-luca-palmieri.md]:
```suggestion
Luca then talked about his book "Zero To Production In Rust: An Introduction To
Backend Development" with which he aimed to bridge the gap between
```

see https://www.zero2prod.com/index.html

File: src/posts/2023-06-28-present-and-future-of-rust-with-luca-palmieri.md (hunk 2 above)
Comment (marcoow) [inline src/posts/2023-06-28-present-and-future-of-rust-with-luca-palmieri.md]:
```suggestion
<iframe width="560" height="315" src="https://youtu.be/Xb7NokhAVKI" title="Embedded video of Luca's interview" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

If you're looking to adopt Rust to develop web backends and need guidance along the way, [get in touch](/contact/) We offer [team augmentation, mentoring, as well as development with Rust](/rust-consulting/).
```

**Review by marcoow — CHANGES_REQUESTED** [review]

File: src/posts/2023-06-28-present-and-future-of-rust-with-luca-palmieri.md (hunk 2 above)
Comment (emmasofiah2o) [reply src/posts/2023-06-28-present-and-future-of-rust-with-luca-palmieri.md]: Yeah, I've commented above. The link is correct, just the iframe doesn't work, but I copied it from Chris and not sure how to fix it

File: src/posts/2023-06-28-present-and-future-of-rust-with-luca-palmieri.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+range of applications. As Rust continues to integrate into libraries and
+frameworks across ecosystems, developers increasingly reap the benefits of its
+power. Whether seeking predictable performance, improved scalability, or reduced
+infrastructure costs, Rust is a versatile language to consider when building
+high-performance and robust software solutions.
+
+Check out the full interview below and don’t hesitate to contact us to learn how
+your company could benefit from adopting Rust.
+
+<iframe width="560" height="315" src="https://youtu.be/Xb7NokhAVKI" title="Embedded video of Luca's interview" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (BobrImperator) [inline src/posts/2023-06-28-present-and-future-of-rust-with-luca-palmieri.md]:
```suggestion
<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/Xb7NokhAVKI?rel=0" title="Embedded video of Luca's interview" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

**Review by marcoow — CHANGES_REQUESTED** [review]
All looks good but let's use a visual that has Luca's photo instead of the generic Rust logo

**Comment by emmasofiah2o** [comment]
@davidhajba what should we use?

**Comment by davidhajba** [comment]
[HERE](https://www.sketch.com/s/066e539c-c206-4261-aafb-9634ab577579/a/y9QYmWz) is the blog post cover with Luca's photo instead of the generic Rust logo

**Comment by emmasofiah2o** [comment]
@marcoow can I merge?

**Review by marcoow — APPROVED** [review]

**Pull request merged**